dpppt-backend-sdk/dpppt-backend-sdk-ws/src/main/java/org/dpppt/backend/sdk/ws/insertmanager/README.md

```diff
@@ -0,0 +1,134 @@
+    manager.addFilter(new RollingStartNumberAfterDayAfterTomorrowFilter());
+    manager.addFilter(new RollingStartNumberInRetentionPeriodFilter(gaenValidationUtils()));
+    manager.addFilter(new NonFakeKeysFilter());
+    manager.addFilter(new ValidRollingPeriodFilter());
+    return manager;
+  }
+}
+```
+
+- `Base64Filter`
+    > This filter validates that the key actually is a correctly encoded base64 string. Since we are using 16 bytes of key data, those can be represented with exactly 24 characters. The validation of the length is already done during model validation and is assumed to be correct when reaching the filter. This filter _throws_ a `KeyIsNotBase64Exception` if any of the keys is wrongly encoded. Every key submitted _MUST_ have correct base64 encoding
+- `KeysMatchingJWTFilter`: 
+    > This filter compares the supplied keys with information found in the JWT token. During the `exposed` request, the onset date, which will be set by the health authority and inserted as a claim into the JWT is the lower bound for allowed key dates. For the `exposednextday` the JWT contains the previously submitted and checked `delayedKeyDate`, which is compared to the actual supplied key.
+- `RollingStartNumberAfterDayAfterTomorrowFilter`: 
+    > Representing the maximum allowed time skew. Any key which is further in the future as the day after tomorrow is considered to be _maliciously_ or faulty (for example because of wrong date time settings) uploaded and is hence filtered out.
+- `RollingStartNumberInRetentionPeriodFilter`: 
+    > Only keys with key date in the configured retention period are inserted into the datbase. Any key which was valid earlier than `RetentionPeriod` is considered to be outdated and not saved in the database. The key would be removed during the next database clean anyways.
+- `NonFakeKeysFilter`
+    > Only keys that are non-fake are inserted into the database, more precicely keys that have the fake flag set to `0`.
+- `ValidRollingPeriodFilter`: 
+    > The `RollingPeriod` represents the 10 minutes interval of the key's validity. Negative numbers are not possible, hence any key having a negative rolling period is considered to be _maliciously_ uploaded. Further, according to [Apple/Googles documentation](https://github.com/google/exposure-notifications-server/blob/main/docs/server_functional_requirements.md) values must be in [1..144]
+
+
+## Additonal Modifiers
+- `IOSLegacyProblemRPLT144FModifier`
+    > This modifier makes sure, that rolling period is always set to 144. Default value according to EN is 144, so just set it to that. This allows to check for the Google-TEKs also on iOS. Because the Rolling Proximity Identifier is based on the TEK and the unix epoch, this should work. The only downside is that iOS will not be able to optimize verification of the TEKs, because it will have to consider each TEK for a whole day.
```

Comment:
as in the class documentation: please add that the other downside of enabling this filter is that keys are delayed, even for clients that would support a rolling period < 144. Maybe also add that this filter is by default not enabled.